getting-started/sandbox-quick-start-guide.md

```diff
@@ -0,0 +1,439 @@
+# INTRODUCTION
+
+**Cloud File Transfer (CFT)** [sandbox][glossary.md] is a test environment that is almost identical to our production environment except that it supports only internet to internet transfers.
+
+In the following sections you will learn how to use APIs in the sandbox to send and receive files. \(Includes resource information on each of the API endpoints, request headers, response schema, sample request and response payloads!\)
```

Comment:
"In the following sections, you will learn how to use our APIs in the CFT sandbox to send and receive files."

added comma, "our" and "CFT" before sandbox, does CFT sandbox make sense here? Wanted to add a little more product id here.